rfb/rfbclient.h

```diff
@@ -123,6 +123,7 @@ typedef struct {
   rfbBool viewOnly;
 
   const char* encodingsString;
+  uint32_t clipboardCap; /*extended clipboard pseudo-encoding cap*/
```

Comment:
Why does this need to be configurable?

Reply:
pseudo clipboard encoding is not a simple feature, it contain many sub-feature and each in-turn combined each other. and it reuse the message format(dont add its new message). these make it not suitable as othe pseudo encoding. it need a per-client-configuration both in init or runing stage.

Reply:
Can you please the consequences in more detail? I don't quite get the point, sorry.

Reply:
this pseudo encoding is really not a normal pseudo encoding. it make client to replace the old clipboard. itself contain 0-text/1-rtf/.../4-file with request/peek/notify/provide. these  make  it suitable/need a configuration(as bitset). currently, we  only support text with notify/provide, which is enough for utf-8 clipboard(it is all we need really...but  we  dont define the spec....). A bitset configuration make we easy to expand (if  need) or change behavior.

Reply:
@wuhanck But can't we just use the UTF8 clipboad impl when the server supports it and fall back to the ISO one if not?

Reply:
Alright, to answer my question: https://stackoverflow.com/a/7048780/361413 this means we could have the situation where the client _app_ using the lib only handles Latin-1 and would be confused by UTF-8 coming in.